packages/odata-common/src/request-builder/request-builder-base.ts

```diff
@@ -48,6 +48,11 @@ export abstract class MethodRequestBuilder<
     return this;
   }
 
+  timeout(timeout: number): this {
```

Comment:
Please consider adding a doc comment and also using a verb in the function name.